app/vmctl/flags.go

```diff
@@ -10,8 +10,9 @@ import (
 )
 
 const (
-	globalSilent  = "s"
-	globalVerbose = "verbose"
+	globalSilent             = "s"
```

Comment:
I noticed that some packages suppress progress bar if `s` is set. This looks incorrect. Let's put an explicit description for these flags:
* s - suppresses all prompts
* verbose - enables additional log messages
* disable-progress-bar - disables the progress bar during migration

This logic defined above need to be true for all migration methods.

Reply:
Disable progress bar only if `disable-progress-bar` flag is used